src/libs/ControlSelection/types.ts

```diff
@@ -1,10 +1,10 @@
-import type CustomRefObject from '@src/types/utils/CustomRefObject';
+type SelectionElement<T> = T & {onselectstart: () => boolean};
 
 type ControlSelectionModule = {
     block: () => void;
     unblock: () => void;
-    blockElement: <T>(ref?: CustomRefObject<T> | null) => void;
-    unblockElement: <T>(ref?: CustomRefObject<T> | null) => void;
+    blockElement: <T>(element?: SelectionElement<T> | null) => void;
+    unblockElement: <T>(element?: SelectionElement<T> | null) => void;
```

Comment:
```suggestion
    blockElement: (element?: HTMLElement | null) => void;
    unblockElement: (element?: HTMLElement | null) => void;
```

What about using `HTMLElement` since this lib is designed for web and this type has `onselectstart`?

Reply:
Done here: https://github.com/Expensify/App/pull/34238/commits/e9a397960ca982ae30b5ca0d17442200e62f19b8